### PR TITLE
Maps links open in the same tab

### PR DESCRIPTION
**Pull request by DanielHeath**

### CODE REVIEW

**Comment by danni** [comment]
++

**Comment by danni** [comment]
Of course, if the user now hits back, the app will break, because it will have lost context.

**Comment by danni** [comment]
The other option is to just add a `title` or `aria-label` attribute "Opens in new tab"

**Comment by DanielHeath** [comment]
Yeah - not 100% sold on writing our own browser-history wrapper.

**Comment by DanielHeath** [comment]
What context will be missing? SessionStore will still be there, right?

**Comment by DanielHeath** [comment]
I've switched back to opening a new tab and adding an `aria-label`

File: src/components/Address.js [hunk 1]
```diff
@@ -48,6 +48,7 @@ class Address extends React.Component {
                 </ScreenReader>
                 <a
                     target="_blank"
+                    aria-label="Open google maps in a new tab"
```
Comment (danni) [inline src/components/Address.js]: Google Maps is a proper noun.

**Comment by danni** [comment]
Otherwise ++

**Pull request merged**